app/oauth/login.js

```diff
@@ -6,10 +6,10 @@ const hydra = require('../lib/hydra')
 const url = require('url')
 
 function getLogin (app) {
-  return async function login(req, res, next) {
+  return async function login (req, res, next) {
     const query = url.parse(req.url, true).query
     const challenge = query.login_challenge
-    if (!challenge) return next(e)
+    if (!challenge) return next()
```

Comment:
`e` was undefined. Should we be defining an error here and passing that to `next`?

Reply:
Not sure, but maybe we should mark it as a TODO?

Reply:
we'll make an issue for this